server/index.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express')
+const path = require('path')
+const querystring = require('querystring')
+const request = require('request')
+const { MongoClient } = require('mongodb')
+require('dotenv/config')
+
+const clientId = 'ee3918a44251433a87cbc842f68bc29f'
```

Comment:
If this information is supposed to be private, it is now public. You should update your credentials with Spotify ASAP and put these in a `.env` file 😢